xbmc/addons/AddonInstaller.h

```diff
@@ -191,12 +191,13 @@ class CAddonInstallJob : public CFileOperationJob
 class CAddonUnInstallJob : public CFileOperationJob
 {
 public:
-  CAddonUnInstallJob(const ADDON::AddonPtr &addon);
+  CAddonUnInstallJob(const ADDON::AddonPtr &addon, bool removeData);
 
   virtual bool DoWork();
 
 private:
   void ClearFavourites();
 
   ADDON::AddonPtr m_addon;
+  bool m_remove; //!< True to remove add-on userdata
```

Comment:
thanks for you annoying and useless comment. i have javafied the shit out of the variable name.